application/controllers/EnrollmentForm.php

```diff
@@ -83,8 +83,8 @@ public function makeStripePayment()
         $this->load->model('Verification_Model');
 
         // Receive data from form, method=POST
-        $data['name'] = $this->input->post('name');
         $data['email'] = $this->input->post('email');
+        [$data['name'], $data['upi'], $data['uid']] = $this->Verification_Model->getUserInfo($data["email"]);
```

Comment:
Can we just get `name` and `upi`? Going forwards, we're probably never going to use their `uid` for anything so we can get rid of the column